docs/getting-started/locally/project.md

```diff
@@ -0,0 +1,10 @@
+# Create a Project
+
+Now that the nio binary is installed, the next step is to create a nio project. A project folder is a way to organize and maintain the blocks, variables, and configurations that you will use in an instance of nio. Using the nio CLI is the easiest way to create a new nio project.
+
+## Create a new nio Project Using the nio CLI
+To create a project template using the CLI, enter the following command:
+
+`nio new first_project`
+
+This will create a `first_project` directory in your working directory that contains your nio project.
```

Comment:
this command does a bunch of stuff—somewhere in the docs we should say what it does, probably in the CLI section